docs/src/project.md

```diff
@@ -22,7 +22,7 @@ Demo
     └── runtests.jl
 ```
 
-usually it's more convenient to create a Julia project via [IonCLI](https://github.com/Roger-luo/IonCLI.jl), after you install IonCLI, you can type the following command in your terminal
+usually it's more convenient to create a Julia project via [Ion](https://github.com/Roger-luo/ion), after you install Ion, you can type the following command in your terminal
 
 ```sh
 ion create Demo --template=comonicon
```

Comment:
```suggestion
ion new Demo
```
or
```suggestion
ion new Demo --template=comonicon
```
?

I'm new to Ion(CLI), and I couldn't find the way to use the comonicon template.

Reply:
Ah I haven't implemented that template yet. You can find all the templates here https://github.com/Roger-luo/ion-templates

Reply:
Should I implement the `comonicon` template first?

It seems the template format has been changed, and I couldn't find a straightforward way to convert [IonBase.jl/templates/comonicon.toml](https://github.com/Roger-luo/IonBase.jl/blob/3676f86222e3ffb1ea46304467ecea60a58cc564/templates/comonicon.toml)

Reply:
Unfortunately yes. I haven't had the time to update the docs to adapt new Ion convention. If you have the time, yes! You need to add a new component first to generate the `Comonicon.toml` file, you can take https://github.com/Roger-luo/ion-templates/blob/main/components/Project.toml.hbs as an example. Then add a new component like https://github.com/Roger-luo/Ion/blob/main/src/ion/blueprints/components/project_file.rs, then create a new template TOML file.

Reply:
@Roger-luo Having just hit this issue and gone through the broken link to find and install rust and then ion, to get all the way to running the command and finding out that it's not called `create` anymore and it's just called `new`, and then hitting a kernel panic to make it to this comment section, I would really recommend that you remove this link and the command until such a time as someone implements it.

Even then, consider instead adding a command in comonicon called "create" that does the thing with the software most users already have installed?

Reply:
I'm happy to file a PR to remove it, if you'd like.

Reply:
Sure please go ahead! Thanks!